labelbox/schema/annotation_import.py

```diff
@@ -0,0 +1,294 @@
+    def _create_from_bytes(cls, client, parent_id, name, bytes_data,
+                           content_len):
+        file_name = cls._make_file_name(parent_id, name)
+        file_args = """filePayload: {
+                file: $file,
+                contentLength: $contentLength
+            }"""
+        query_str = cls._build_import_predictions_query(
+            file_args, "$file: Upload!, $contentLength: Int!")
+        variables = {
+            "file": None,
+            "contentLength": content_len,
+            "parent_id": parent_id,
+            "name": name,
+            "predictionType": cls.import_type.value
+        }
+        operations = json.dumps({"variables": variables, "query": query_str})
+        data = {
+            "operations": operations,
+            "map": (None, json.dumps({file_name: ["variables.file"]}))
+        }
+        file_data = (file_name, bytes_data, NDJSON_MIME_TYPE)
+        files = {file_name: file_data}
+
+        response = client.execute(data=data, files=files)
+        return cls(client, response['createAnnotationImport'])
+
+    @classmethod
+    def _create_from_objects(cls, client, parent_id, name, predictions):
+        data_str = ndjson.dumps(predictions)
+        if not data_str:
+            raise ValueError('annotations cannot be empty')
+        data = data_str.encode('utf-8')
+        return cls._create_from_bytes(client, parent_id, name, data, len(data))
+
+    @classmethod
+    def _create_from_file(cls, client, parent_id, name, path):
+        if os.path.exists(path):
+            with open(path, 'rb') as f:
+                return cls._create_from_bytes(client, parent_id, name, f,
+                                              os.stat(path).st_size)
+        elif requests.head(path):
+            return cls._create_from_url(client, parent_id, name, path)
+        raise ValueError(
+            f"Path {path} is not accessible locally or on a remote server")
+
+    def create_from_objects(*args, **kwargs):
+        raise NotImplementedError("")
+
+    def create_from_file(*args, **kwargs):
+        raise NotImplementedError("")
+
+    def get_parent_id(*args, **kwargs):
+        raise NotImplementedError("")
+
+
+class MEAPredictionImport(AnnotationImport):
+    id_name = "modelRunId"
+    import_type = ImportType.MODEL_ERROR_ANALYSIS
+    model_run_id = Field.String("model_run_id")
+
+    def get_parent_id(self):
+        return self.model_run_id
+
+    @classmethod
+    def create_from_file(cls, client, model_run_id, name, path):
+        return cls._create_from_file(client=client,
+                                     parent_id=model_run_id,
+                                     name=name,
+                                     path=path)
+
+    @classmethod
+    def create_from_objects(cls, client, model_run_id, name, predictions):
+        return cls._create_from_objects(client, model_run_id, name, predictions)
+
+
+class MALPredictionImport(AnnotationImport):
```

Comment:
Im curious why we need this? I dont think were going to be allowing MAL imports via SDK through this mutation for a little while.